api/utils.go

```diff
@@ -82,7 +100,7 @@ func ToBusinessError(err error) *BusinessError {
 			}
 			return &BusinessError{Code: sgtnerror.UnknownError.Code(), HTTPCode: sgtnerror.UnknownError.HTTPCode(), UserMsg: e.Error()}
```

Comment:
why this error can't define in types.go file

Reply:
The user error message is special for each error. 
UserMsg: e.Error()}